tests/test_view/test_views.py

```diff
@@ -393,63 +223,97 @@ def check_token(result):
     assert result["permissions"] == ["full_control"]
 
 
-def test_token(client: TestClient, token: str):
-    assert client.get("/token").status_code == 401
-    resp = client.get("/token", headers={"Authorization": f"Bearer {token}"})
-    assert resp.status_code == 200
-    j = resp.json()
-    assert j["count"] == 1
-    assert not j["previous"]
-    assert not j["next"]
-    result = j["result"]
-    assert isinstance(result, list)
-    assert len(result) == 1
-    result = result[0]
-    check_token(result)
-
-
-def test_token_current(client: TestClient, token: str):
+@Parametrization.autodetect_parameters()
+@Parametrization.case(name="unauthorized", authorized=False)
+@Parametrization.case(name="authorized", authorized=True)
+def test_token(client: TestClient, token_obj, authorized: bool):
```

Comment:
Hmm, in some cases parametrization adds more complexity than needed. Maybe we could stick to non-parametrized tests for those? It's creating unnecessary branches while it just checks unauthorized access.